### PR TITLE
Added setCoGatewayContractInternal method

### PR DESCRIPTION
**Pull request by gulshanvasnani**
PR contains changes to add `setCoGatewayInternal` method in UtilityToken contract.

References https://github.com/OpenST/brandedtoken-contracts/issues/166

### CODE REVIEW

**Comment by schemar** [comment]
Reviewing.

@gulshanvasnani please reference and connect the relevant issue.

File: contracts/UtilityToken.sol [hunk 1]
```diff
@@ -191,6 +169,43 @@ contract UtilityToken is EIP20Token, Organized, UtilityTokenInterface {
...
 
+    /**
+     * @notice Sets the CoGateway contract address. This can be called by
+     *         UtilityToken contract or contract inheriting it. This can be
+     *         set only once.
+     * @param _coGatewayAddress CoGateway contract address
+     *
+     * @return success_ `true` if CoGateway address was set
+     */
+    function setCoGatewayInternal(address _coGateway)
```
Comment (schemar) [inline contracts/UtilityToken.sol]: A function head should be "all or nothing" multi-line.
```suggestion
    function setCoGatewayInternal(
        address _coGateway
    )
```

File: contracts/UtilityToken.sol [hunk 2]
```diff
@@ -191,6 +169,43 @@ contract UtilityToken is EIP20Token, Organized, UtilityTokenInterface {
...
 
+    /**
+     * @notice Sets the CoGateway contract address. This can be called by
+     *         UtilityToken contract or contract inheriting it. This can be
+     *         set only once.
+     * @param _coGatewayAddress CoGateway contract address
+     *
+     * @return success_ `true` if CoGateway address was set
+     */
+    function setCoGatewayInternal(address _coGatewayAddress)
```
Comment (schemar) [inline contracts/UtilityToken.sol]: A function head should be "all or nothing" multi-line:
```suggestion
    function setCoGatewayInternal(
        address _coGatewayAddress
    )
```

**Comment by schemar** [comment]
> @gulshanvasnani please reference and connect the relevant issue.

I did.

**Comment by gulshanvasnani** [comment]
Made the suggested changes.
Now the PR is available for review.

**Pull request merged**